operator/apis/operator/v1alpha1/banyandb_types.go

```diff
@@ -0,0 +1,66 @@
+package v1alpha1
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type BanyanDBSpec struct {
```

Comment:
Affinity is missing

Reply:
We will introduce a `PodTemplate` field to support `Affinity`.

Reply:
@hanahmily should we expose total `PodTemplate` to user?  I found that in OapServer CRD we only expose some key fields like`Image`, while providing default value for other fields. Should we follow this practice?

Reply:
The key to exposing the template is whether the user needs it. We should support the [Affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/) here.

Reply:
Got it